### PR TITLE
Fix broken documentation source links

### PR DESCRIPTION
**Pull request by fxmarty**
See https://github.com/huggingface/optimum/pull/357

### CODE REVIEW

**Review by lewtun — APPROVED** [review]
LGTM - let's get approval from @echarlaix before merging :)

**Review by echarlaix — APPROVED** [review]
LGTM, thanks for taking care of that @fxmarty

**Pull request merged**
